### ISSUE TITLE
Validation set values in individuals

### ISSUE THREAD

**Issue by brunelloandrea** [DEAP/deap#598]
Dear all,

I am currently using DEAP to solve a bi-objective optimization problem.

Other than the 2 values of the objective function, calculated on the training set, I would like to keep track, for each individual, of the same two values, but calculated on a separate validation set, in order to witness possible overfitting effects.

I am currently using a hand-made list to do that, separated from the individuals.
Is there a way to handle such values within an individual, in the same way as it is done for the fitness values?

That would simplify things a lot, especially for what concerns updating the training/validation values only for the individuals that have undergone crossover or mutation.

Thanks,
Andrea

**Comment by fmder** [comment]
You could add a new attribute at individual creation.
```python
creator.create("individual", list, fitness=creator.FitnessMin, valid_fitness=creator.FitnessMin)"
```

And then you'd have to add, in the algorithm, when assigning fitness
```python
ind.fitness.values = 1,   # your train fitness
ind.valid_fitness.values = 0.7,   # your valid fitness
```

**Comment by brunelloandrea** [comment]
Perfect, thank you!
